.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Description
+
+* Quickly summarize your changes in a few bullet points (can and should correspond to [CHANGELOG.md](https://github.com/fkie-cad/Logprep/blob/main/CHANGELOG.md))
+* Fixes #XXX (insert issue number here), if there is a corresponding GH issue
+
+## Assignee
+- [ ] The changes adhere to the [contribution guidelines](https://github.com/fkie-cad/Logprep/blob/main/CONTRIBUTING.md)
+- [ ] I have performed a self-review of my code
+- [ ] My changes generate no new warnings
```

Comment:
Thank you, the pull request looks good to me. I have only one small remark.
Maybe we could clarify this, since the information contradicts the contribution guidelines:

```suggestion
- [ ] My changes generate no new warnings other than deprecation warnings
```